ddtrace/internal/remoteconfig/client.py

```diff
@@ -199,6 +203,12 @@ def __init__(self):
             if container_id is not None:
                 self._headers["Datadog-Container-Id"] = container_id
 
+        tags = global_config.tags.copy()
```

Comment:
Looking at the code it seems the DD_TAGS configuration is used to set config.tags and then config.tags is used to set tracer._tags. Since both tracer._tags and config.tags are mutable, I can see a scenario config.tags and tracer.tags contain different values. In this case which one should be used in remote config?

```suggestion
        tags = ddtrace.tracer.get_tags()
```

Reply:
That's a good question. I'm assuming the tracer will mutate the tags from the `DD_TAGS` variable for its own uses. I think we want to use whatever is passed as `DD_TAGS` here. Hopefully in the future we will have a cleaner separation between global configuration, and product-specific one 🙂

Reply:
Ah and `_tracer` here refers to the tracer library, not the tracer product 😕 .